src/main/java/com/ays/common/scheduler/UnusedInvalidTokenDeletionScheduler.java

```diff
@@ -0,0 +1,36 @@
+package com.ays.common.scheduler;
+
+import com.ays.auth.repository.AysInvalidTokenRepository;
+import com.ays.parameter.model.AysParameter;
+import com.ays.parameter.service.AysParameterService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+/**
+ * Scheduler to remove all unused invalid token.
+ */
+@Component
+@RequiredArgsConstructor
+@ConditionalOnProperty(name = "ays.scheduler.UnusedInvalidTokenDeletionScheduler.enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
Neden conditional bir parametreye bağladık?

Reply:
Eğer uygulama ECS ile birden fazla instance ayağa kalktığında birden fazla scheduler çalışabilir. İlerde bu property'i sadece bir instance'da çalışması için kullanabiliriz diye düşündüm. Eğer bu yol çözmez ise bu sorunu düşünmemiz gerekicek.

Reply:
Toplantı yapıp bu kısımları değerlendirelim, neler yapabileceğimize bakalım

Reply:
Burada `matchIfMissing` kısmındaki default değeri `false` yapabiliriz ya da default değer `false` ise hiç yazmayabiliriz

Reply:
` matchIfMissing` değerini ` ays.scheduler.UnusedInvalidTokenDeletionScheduler.enabled` olmadığında default olarak true yapacak. Burada false olma durumu var mı? @ibrahimdenizz

Reply:
@Rapter1990 ECS'te birden fazla instance çalışırsa sadece birinde enabled yapmamız gerekebilir.